packages/@aws-cdk/aws-lambda-python/lib/layer.ts

```diff
@@ -0,0 +1,54 @@
+import * as path from 'path';
+import * as lambda from '@aws-cdk/aws-lambda';
+import * as cdk from '@aws-cdk/core';
+import { bundle } from './bundling';
+
+/**
+ * Properties for PythonLayerVersion
+ */
+export interface PythonLayerVersionProps extends lambda.LayerVersionOptions {
+  /**
+   * The path to the root directory of the lambda layer.
+   */
+  readonly entry: string;
+
+  /**
+   * The runtimes compatible with the python layer.
+   *
+   * @default - All runtimes are supported.
+   */
+  readonly compatibleRuntimes?: lambda.Runtime[];
```

Comment:
See [comment](https://github.com/aws/aws-cdk/pull/9582/files?file-filters%5B%5D=.dependencies&file-filters%5B%5D=.ts&file-filters%5B%5D=No+extension&w=1#r480390847) in [lambda/lib/layers.ts](packages/@aws-cdk/aws-lambda/lib/layers.ts)